### PR TITLE
[8.0.x] example: Add Apply

### PR DESCRIPTION
**Pull request by NeQuissimus**

### CODE REVIEW

File: example/src/main/tut/typeclass/Apply.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+layout: docs
+title:  "Apply"
+---
+
+# Apply
+
+*A strong lax semi-monoidal endofunctor. This is equivalent to an Applicative without pure.*
```
Comment (hrhino) [inline example/src/main/tut/typeclass/Apply.md]: If you make an `Applicative.md`, please try to remember to link it here.

File: example/src/main/tut/typeclass/Apply.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+In addition to the laws of a [Functor](./Functor.html), an Apply instance needs to satisfy the following law:
+
+- Associativity: `x.ap(y).ap(z) === x.ap(y.ap(z)).map { case (x, (y, z)) => ((x, y), z) }`
```
Comment (hrhino) [inline example/src/main/tut/typeclass/Apply.md]: Is this the law? It looks like you're talking in terms of `product` or `tuple2` or other such `Apply`-based functions.

File: example/src/main/tut/typeclass/Apply.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+```tut:silent
+import scalaz.Prelude._
+```
+
+## Instance declaration
+
+```tut
+implicit val listApply: Apply[List] = new Apply[List] {
+  val functor = implicitly
```
Comment (hrhino) [inline example/src/main/tut/typeclass/Apply.md]: do we have an `ApplyClass`?

File: example/src/main/tut/typeclass/Apply.md (hunk 2 above)
Comment (NeQuissimus) [reply example/src/main/tut/typeclass/Apply.md]: Oh, did I mess this up? I'll look into it.

**Comment by NeQuissimus** [comment]
I updated the examples but I ask that they be checked for correctness. I really would not want to have documentation that is misleading :)
And `Apply` has been a new one for me...

**Comment by NeQuissimus** [comment]
#1554 updates Ruby and Jekyll, which is failing the build here

**Comment by NeQuissimus** [comment]
Updated with upstream changes

**Pull request merged**